csaf_files/IT/white/2024/va-24-317-01.json

```diff
@@ -0,0 +1,898 @@
+        "title": "Critical Infrastructure Sectors",
+        "category": "description"
+      }
+    ],
+    "references": [
+      {
+        "url": "https://raw.githubusercontent.com/cisagov/CSAF/develop/csaf_files/IT/white/2024/va-24-317-01.json",
+        "summary": "Vulnerability Advisory VA-24-317-01 CSAF",
+        "category": "self"
+      }
+    ]
+  },
+  "product_tree": {
+    "branches": [
+      {
+        "category": "vendor",
+        "name": "Ivanti",
+        "branches": [
+          {
+            "category": "product_name",
+            "name": "Connect Secure",
+            "branches": [
+              {
+                "category": "product_version_range",
+                "name": "0 < 22.6R2",
+                "product": {
+                  "name": "Ivanti Connect Secure 0 < 22.6R2",
+                  "product_id": "CSAFPID-0001"
+                }
+              },
+              {
+                "category": "product_version",
+                "name": "22.6R2",
+                "product": {
+                  "name": "Ivanti Connect Secure 22.6R2",
+                  "product_id": "CSAFPID-0002"
+                }
+              },
+              {
+                "category": "product_version_range",
+                "name": "0 < 9.1R18.7",
+                "product": {
+                  "name": "Ivanti Connect Secure 0 < 9.1R18.7",
+                  "product_id": "CSAFPID-0003"
+                }
+              },
+              {
+                "category": "product_version",
+                "name": "9.1R18.7",
+                "product": {
+                  "name": "Ivanti Connect Secure 9.1R18.7",
+                  "product_id": "CSAFPID-0004"
+                }
+              },
+              {
+                "category": "product_version_range",
+                "name": "0 < 22.7R2.1",
+                "product": {
+                  "name": "Ivanti Connect Secure 0 < 22.7R2.1",
+                  "product_id": "CSAFPID-0007"
+                }
+              },
+              {
+                "category": "product_version",
+                "name": "22.7R2.1",
+                "product": {
+                  "name": "Ivanti Connect Secure 22.7R2.1",
+                  "product_id": "CSAFPID-0008"
+                }
+              },
+              {
+                "category": "product_version_range",
+                "name": "0 < 9.1R18.9",
+                "product": {
+                  "name": "Ivanti Connect Secure 0 < 9.1R18.9",
+                  "product_id": "CSAFPID-0009"
+                }
+              },
+              {
+                "category": "product_version",
+                "name": "9.1R18.9",
+                "product": {
+                  "name": "Ivanti Connect Secure 9.1R18.9",
+                  "product_id": "CSAFPID-0010"
+                }
+              },
+              {
+                "category": "product_version_range",
+                "name": "0 < 22.7R2.2",
```

Comment:
This should be `"<22.7R2.2"`.

See documentation: https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#31232-branches-type---name-under-product-version-range